integration-commons/src/test/java/org/symphonyoss/integration/exception/bootstrap/CertificateNotFoundExceptionTest.java

```diff
@@ -0,0 +1,53 @@
+package org.symphonyoss.integration.exception.bootstrap;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link CertificateNotFoundException}.
+ * Created by crepache on 23/06/17.
+ */
+public class CertificateNotFoundExceptionTest {
+
+  @Test
+  public void testCertificateNotFoundException() {
+    CertificateNotFoundException exception = new CertificateNotFoundException();
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.